{"name": "test_add_member_fail[user2-11528872802-手机已存在于通讯录，无法添加]", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'click'", "trace": "self = <testcases.test_addMember.TestAddMember object at 0x000002BB0FC1D6D8>\nuser = 'user2', phone = '11528872802', expect = '手机已存在于通讯录，无法添加'\n\n    @pytest.mark.parametrize(\"user,phone,expect\", test_data['test_add_member_fail'])\n    def test_add_member_fail(self, user, phone, expect):\n>       fail_text = self.addMember = self.app.main().go_to_address().go_to_add_member().add_member_fail(user, phone)\n\ntestcases\\test_addMember.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage\\main_page.py:14: in go_to_address\n    self.steps(os.getcwd() + \"/pageYaml/main_page.yaml\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <page.main_page.Main object at 0x000002BB0FD32E10>\npath = 'E:\\\\document\\\\code\\\\pythonCode\\\\appium_ui_testFramework/pageYaml/main_page.yaml'\n\n    def steps(self, path):\n        name = inspect.stack()[1].function\n        with open(path, encoding=\"utf-8\") as f:\n            dat = yaml.safe_load(f)\n            steps = dat[name]\n            for step in steps:\n                raw = json.dumps(step)\n                for key, value in self._param.items():\n                    raw = raw.replace(f'${{{key}}}', value)\n                step = json.loads(raw)\n                if \"action\" in step.keys():\n                    if step[\"action\"] == \"click\":\n>                       self.find(step['by'], step['locator']).click()\nE                       AttributeError: 'NoneType' object has no attribute 'click'\n\npage\\base_page.py:64: AttributeError"}, "attachments": [{"source": "060c5f0f-082d-40d4-b32d-3ba28efe4481-attachment.png", "type": "image/png"}, {"name": "log", "source": "0922930d-a9d0-4397-a5e2-410710a45832-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "'user2'"}, {"name": "phone", "value": "'11528872802'"}, {"name": "expect", "value": "'手机已存在于通讯录，无法添加'"}], "start": 1626592783968, "stop": 1626592791793, "uuid": "d379cab1-ce52-4e07-ac2a-d3ca1a42026b", "historyId": "f072b275c56a30d0c23d296fd0a67095", "testCaseId": "b251c483add15ec609094c40b1ea12dc", "fullName": "testcases.test_addMember.TestAddMember#test_add_member_fail", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_addMember"}, {"name": "subSuite", "value": "TestAddMember"}, {"name": "host", "value": "z"}, {"name": "thread", "value": "21348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_addMember"}]}